### ISSUE TITLE
Drop our wrapper for pytest

### ISSUE THREAD

**Issue by nyh** [scylladb/scylladb#12412]
When Fedora 37 came out and we started to use it in our dbuild, we discovered that the "pytest" script started to run Python with the "-s" option. As a result, the test code can only use official Python packages installed by root, and doesn't see the user's personally installed packages with pip and so on. This caused problems for Scylla developers who install the Cassandra or Scylla driver locally and want to run tests using that driver.

So to fix that, we added in commit e5e7780f32d20a328c8981ae80e2f080323482c4 a script `test/pytest` which runs pytest without that "-s" option, and started to use this test/pytest instead of the system-install "pytest".

But the use of "Python -s" was a Fedora bug. I reported it in https://bugzilla.redhat.com/show_bug.cgi?id=2152171 and they quickly acknowledged that it was unintentional and fixed it - and the fix is now in Fedora 37. In pull request https://github.com/scylladb/scylladb/pull/12397 we will update dbuild to include (among other things) this fix, and which point we can drop the silly "pytest" wrapper.

Basically we can delete the `test/pytest` wrapper, and the calls to test/pytest in:
1. `test.py` (two places)
2. `test/cql-pytest/run.py`

**Comment by avikivity** [comment]
Not a bug, not backporting.
